APMrover2/mode_loiter.cpp

```diff
@@ -24,13 +24,15 @@ void ModeLoiter::update()
     _distance_to_destination = get_distance(rover.current_loc, _destination);
 
     // if within waypoint radius slew desired speed towards zero and use existing desired heading
-    if (_distance_to_destination <= g.waypoint_radius) {
-        _desired_speed = attitude_control.get_desired_speed_accel_limited(0.0f, rover.G_Dt);
+    if (_distance_to_destination <= g2.loit_radius) {
+        // sailboats do not stop
+        const float desired_speed_within_radius = g2.motors.has_sail() ? 0.1f : 0.0f;
```

Comment:
shouldn't this loiter speed be a parameter ?

Reply:
currently sailboats do not do speed control, they either try and sail at full speed or not. So the value makes no difference, it just has to be above zero.  Hopefully we can add some speed control in the future.